### PR TITLE
throws twenty gbp at that goodboyless man (ATH1909) who i walk past on my way to work every morning

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

What will they do with their 20 GBP? Will they build it up with more fixes? Or will they fiend on features and rebalances until they find themselves cemented in the negative GBP again?

## Why It's Good For The Game

image

### CODE REVIEW

File: .github/gbp-balances.toml [hunk 1]
```diff
@@ -498,7 +498,7 @@
 42397676 = 6 # jlsnow301
 42542238 = 373 # Ghommie
 42553659 = 32 # Surrealaser
-42606352 = -6 # ATH1909
+42606352 = 16 # ATH1909
```
Comment (Fikou) [inline .github/gbp-balances.toml:501]: -6+20 is 14 you dumb fucking idiot
this is what american education system teaches

Comment (tralezab) [reply .github/gbp-balances.toml:501]: american educuhgatuion

**Comment by tralezab** [comment]
nobody expand that review fikou made i dealt with it

**Comment by necromanceranne** [comment]
This feels wrong.

**Comment by LemonInTheDark** [comment]
No

**Comment by MMMiracles** [comment]
image
hand over your gbp

**Pull request closed without merging**